src/extensions/default/QuickView/main.js

```diff
@@ -325,6 +325,11 @@ define(function (require, exports, module) {
         function hasLengthInPixels(args) {
             return (args.length > 1 && args[1].indexOf("px") > 0);
         }
+        
+        // Ensures that input is in usable hex format
+            function ensureHexFormat(str) {
```

Comment:
@nyteksf 
wrong indentation, do you have `brackets-eslint` extension installed? it should have given you a warning here